readme.md

```diff
@@ -29,9 +29,9 @@ snakecaseKeys({'foo-bar': true, nested: {fooBaz: 'bar'}});
 ##### obj
 
 *Required*  
-Type: `object`
```

Comment:
This is an informal pseudo-JS notation. I am not trying to use TypeScript types here. See `exclude` for example.